### PR TITLE
Changing UI for sketch

### LINKED ISSUE #630: Changing UI for sketch
image
Moving the current pannel completely to the right so that it doesn't feel like the complete copy.

### PR DESCRIPTION
**Pull request by SalmaFarghaly**
closes #630 

Changes:

- Re-added the canvas color
- Fixed the scroll option on sketch page
- Reset canvas color value in input field to white after clicking the clear icon.

**Before adding the canvas color:**
beforeBug (1)

**Before resetting the canvas color input to white when user clicks clear** 
There was a bug in this part ,if the user chooses a certain color to be the canvas color and then clicks clear , the background turns to white but the input field of canvas color still holds the old chosen color
beforebug2 (1)

**After**
afterbug (1)

### CODE REVIEW

File: src/pages/Sketch/Sketch.module.css [hunk 1]
```diff
@@ -7,5 +7,5 @@
     align-items: center;
     position: relative;
     height: calc(100vh - 100px);
-    overflow: hidden;
+    /* overflow: hidden; */
```
Comment (iamdarshshah) [inline src/pages/Sketch/Sketch.module.css:10]:
```suggestion
```
Remove it, if it's not needed anymore.

Comment (SalmaFarghaly) [reply src/pages/Sketch/Sketch.module.css:10]: Okay, Also the toolbox's content will continue to appear outside the canvas , Is that Okay ?
outtoolbox (1)

Comment (iamdarshshah) [reply src/pages/Sketch/Sketch.module.css:10]: > Okay, Also the toolbox's content will continue to appear outside the canvas , Is that Okay ?

@SalmaFarghaly Do you mean to say that if we change the `Canvas Color`, it won't get reflected in the toolbox, isn't it?

Comment (SalmaFarghaly) [reply src/pages/Sketch/Sketch.module.css:10]: No I don't mean that , I mean the "Edge mode" part and the "Undo/Redo" part that are in the toolbox appear below the horizontal light blue line that is in the photo .

**Review by anushbhatia — COMMENTED** [review]
If the categories become too large add them to the side of the current column so that it won't overflow and the user does not need to scroll for it.

**Comment by SalmaFarghaly** [comment]
@anushbhatia Do you mean the toolbox by the current column ?

**Comment by anushbhatia** [comment]
yes

**Comment by SalmaFarghaly** [comment]
@anushbhatia Should I place them outside the toolbox ? I just can't imagine where I will place them Can you explain more?

**Comment by anushbhatia** [comment]
Doc2pen - Sketch and 5 more pages - Personal - Microsoft​ Edge 26-03-2021 02_46_49 (1)_LI

**Comment by SalmaFarghaly** [comment]
I think that will take a space from the sketch area and it is better to have large space in the sketch area.

**Comment by smaranjitghose** [comment]
For the time being the PR fixes a lot of issues which were previosly not taken care of, @SalmaFarghaly could you incorporate the changes suggested above by @iamdarshshah  and tag me.

@anushbhatia feedback regarding the auto resize of the toolbar column is definitely crucial but I believe we can tackle that in a new PR once the current changes are incorporated. Cool?

**Comment by SalmaFarghaly** [comment]
@smaranjitghose Done, I have incorporated the changes.

**Review by iamdarshshah — APPROVED** [review]

**Comment by anushbhatia** [comment]
Feel free to raise the PR for incorporating undo/redo feature in the extra column.

**Comment by SalmaFarghaly** [comment]
@anushbhatia Will I add also the "Edge mode" in the extra column?

**Pull request merged**